quickstart/201-azfw-multi-addresses/main.tf

```diff
@@ -0,0 +1,227 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "random_password" "password" {
+  count = 2
```

Comment:
Please run `terraform fmt` to format this folder, thanks.

Reply:
done